Model/FileIndex.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace SomethingDigital\UpgradeHelper\Model;
+
+class FileIndex
+{
+    const THEME_OVERRIDE = 'theme_override';
+
+    const MODULE_OVERRIDE = 'module_override';
```

Comment:
I don't like these names as they're not necessarily overrides...it's just an index of all files in `app/code`, `app/design` and `vendor/`. Can we just call them `THEME_FILES` and `MODULE_FILES`?